src/Raven.Client/ServerWide/Tcp/TcpConnectionHeader.cs

```diff
@@ -16,5 +18,15 @@ public enum OperationTypes
         public string SourceNodeTag { get; set; }
 
         public OperationTypes Operation { get; set; }
+
+        public int OperationVersion { get; set; }
+
+        public static readonly Dictionary<OperationTypes, int> TcpVersions = new Dictionary<OperationTypes, int>
+        {
```

Comment:
How is it better?
Nobody should ever set this with a number, they will use the dictionary value.
Having 4 different properties is more error prone.